adflow/pyADflow.py

```diff
@@ -4762,6 +4781,8 @@ def _getDefaultOptions():
             "windAxis": [bool, False],
             "alphaFollowing": [bool, True],
             "TSStability": [bool, False],
+            "useTSInterpolatedGridVelocity": [bool, False],
```

Comment:
Also, minor point but the case in options.yaml must match the case here, otherwise the docs build will not work.